pkg/subscriptions/index.html

```diff
@@ -145,6 +145,17 @@ <h4 class="modal-title" translatable="yes">Register system</h4>
                   <span id="subscriptions-register-password-note-details"></span>
                 </td>
               </tr>
+              <tr>
+                <td class="top">
+                  <label class="control-label" for="subscription-register-org" translatable="yes">
+                    Organization
+                  </label>
+                </td>
+                <td>
+                  <input id="subscription-register-org" class="form-control" type="text" placeholder="used in multi-tenant setups">
```

Comment:
> If an information field is needed it should be an info popover. @andreasn What do you think?

should I remove that info? Usually people will have that, if they want to register a system. It's an entry required for very specific setups and part of the the credentials, basically.